server/controllers/column.js

```diff
@@ -1,91 +1,102 @@
 const User = require("../models/User");
 const asyncHandler = require("express-async-handler");
-const { v4: uuidv4 } = require("uuid");
+const ObjectID = require("mongodb").ObjectID;
```

Comment:
We can do middle-ware validation to ensure mongo ids are being passed into the url parameters, let's add in middleware validation please.

Reply:
I have already removed those UUIDs, please you can check the branch.